### ISSUE TITLE
18.11.4.9: get_constant not actually a UnaryTypeTrait

### ISSUE THREAD

**Issue by hubert-reinterpretcast** [cplusplus/reflection-ts#82]
A `constexpr` variable may be of class type, so inheriting from `integral_constant` would not provide a `value` in the manner described (since non-type template parameters of class type are rather newer than C++14).

**Comment by camio** [comment]
Sounds like a DR no?

**Comment by hubert-reinterpretcast** [comment]
I guess more words may be necessary to describe the type if we do not describe by way of `UnaryTypeTrait`. What would be the plan to track this then?

**Comment by Axel-Naumann** [comment]
I agree with David: it looks like a design issue to me; not part of the wording review but an issue that needs to be handled post-publication. Tracking this as a Github issue is fine for me.
